Taskfile.Shared.yml

```
@@ -62,56 +62,31 @@ tasks:
     label: "{{.TASK}}-{{.NAME}}"
     dir: coprocessors/{{.NAME}}
     cmds:
-      - docker run -d --rm --name {{.NAME}}-co --env PORT="{{.COPROCESSOR_PORT}}" -m 1g --cpus=1 -p {{.COPROCESSOR_PORT}}:{{.COPROCESSOR_PORT}} {{.NAME}}-co
+      - docker run -d --rm --name coprocessor --net=task --env PORT="8000" -p 8000:8000 -m 1g --cpus=1 {{.NAME}}-co
     vars:
       NAME: "{{.NAME|default .CLI_ARGS}}"
   cleanup-coprocessor:
     label: "{{.TASK}}-{{.NAME}}"
     cmds:
-      - docker stop {{.NAME}}-co
+      - docker stop coprocessor
     vars:
       NAME: "{{.NAME|default .CLI_ARGS}}"
 
   start-subgraphs:
+    dir: subgraphs/fixed
     cmds:
-      - task: build-subgraph
-        vars:
-          NAME: a
-      - task: build-subgraph
-        vars:
-          NAME: b
-      - task: start-subgraph
-        vars:
-          NAME: a
-          PORT: 8081
-      - task: start-subgraph
-        vars:
-          NAME: b
-          PORT: 8080
+      - DOCKER_BUILDKIT=1 docker build . -t fixed
+      - docker run -d --name subgraph --net=task -m 1g --cpus=1 -p 8082:8082 fixed
```

```suggestion
      - docker run -d  --rm --name subgraph --net=task -p 8082:8082 fixed
```

we can pass the --rm flag to terminate upon closing, and also remove bounds since we're not testing the subgraph, just the router/coprocessor